addons/web_editor/static/src/js/wysiwyg/widgets/media_dialog.js

```diff
@@ -44,145 +52,160 @@ var MediaDialog = WeDialog.extend({
             },
         });
 
-        this.media = media;
-        this.$media = $(media);
-
-        this.multiImages = options.multiImages;
-        var onlyImages = options.onlyImages || this.multiImages;
-        this.noImages = options.noImages;
-        this.noDocuments = onlyImages || options.noDocuments;
-        this.noIcons = onlyImages || options.noIcons;
-        this.noVideos = onlyImages || options.noVideos;
```

Comment:
I didn't know, I just saw they were not used... This makes things a lot more complicated to clean up if I have to take into account potentially broken features and potentially upcoming features too.

Reply:
Well you cannot decide to clean up code without thinking about why it was there in the first place... especially in Odoo. And here, it is no clean up, you just removed a bunch of options that were correctly coded and that the community may already be using.

Reply:
This is an unfortunate fact but you are correct of course, I'll double check everything.

I don't agree with keeping unused options in general, but in this case I can see why it would be ok to have them, so I'll add them back.